packages/harvester/harvester-cloud-provider/generated-changes/patch/Chart.yaml.patch

```diff
@@ -1,20 +1,14 @@
 --- charts-original/Chart.yaml
 +++ charts/Chart.yaml
-@@ -1,13 +1,14 @@
- annotations:
-   catalog.cattle.io/certified: rancher
-   catalog.cattle.io/display-name: Harvester Cloud Provider
--  catalog.cattle.io/kube-version: '>= 1.18.0-0 < 1.24.0-0'
-+  catalog.cattle.io/kube-version: '>= 1.18.0-0 < 1.25.0-0'
+@@ -5,9 +5,10 @@
    catalog.cattle.io/namespace: kube-system
    catalog.cattle.io/os: linux
--  catalog.cattle.io/permits-os: linux
-+  catalog.cattle.io/permits-os: linux,windows
--  catalog.cattle.io/rancher-version: '>= 2.6.1-0 < 2.7.0-0'
-+  catalog.cattle.io/rancher-version: '>= 2.7.0-0 < 2.8.0-0'
+   catalog.cattle.io/permits-os: linux
+-  catalog.cattle.io/rancher-version: '>= 2.6.1-0 < 2.8.0-0'
++  catalog.cattle.io/rancher-version: '>= 2.7-0 < 2.8.0-0'
```

Comment:
Just noticed version constraint has a typo here.
```suggestion
+  catalog.cattle.io/rancher-version: '>= 2.7.0-0 < 2.8.0-0'
```

Reply:
Hi @snasovich, nice catch. Thanks for your review!
Already fixed it. Please help to review it again.